Tasks/VsTest/vstest.ts

```diff
@@ -432,16 +442,17 @@ function setRunInParallellIfApplicable(vsVersion: number) {
     if (runInParallel) {
         if (!isNaN(vsVersion) && vsVersion >= 14) {
             var vs14Common = tl.getVariable("VS140COMNTools");
-            if ((vs14Common && pathExistsAsFile(path.join(vs14Common, "..\\IDE\\CommonExtensions\\Microsoft\\TestWindow\\TE.TestModes.dll"))) || vsVersion > 14) {
+            if (vsVersion > 14 || (vs14Common && pathExistsAsFile(path.join(vs14Common, "..\\IDE\\CommonExtensions\\Microsoft\\TestWindow\\TE.TestModes.dll")))) {
```

Comment:
and we r sure it wont be removed going forward? we cant guarantee that correct?

---
In reply to: [67283885](https://github.com/Microsoft/vsts-tasks/pull/1886#discussion_r67283885) [](ancestors = 67283885)